packages/components/src/components/typography/Link/index.tsx

```diff
@@ -56,7 +56,17 @@ interface Props {
 }
 
 const Link = ({ icon, iconProps, ...props }: Props) => (
-    <A href={props.href} target={props.target || '_blank'} rel="noreferrer noopener" {...props}>
+    <A
+        href={props.href}
+        target={props.target || '_blank'}
+        rel="noreferrer noopener"
+        {...props}
+        onClick={(e: any) => {
```

Comment:
I would rather to see a proper type here (hover on onClick before declaring own func will guide you)